src/main/java/eu/adlogix/utils/AdlFileUtils.java

```diff
@@ -0,0 +1,52 @@
+package eu.adlogix.utils;
+
```

Comment:
This is a copy-paste code from the java utils project. I didn't change the package name and kept the original package name. Shall I keep it this way or move to an appnexus package?